schemas/dab.draft.schema.json

```
@@ -261,6 +261,28 @@
             }
           }
         },
+        "log-level": {
+          "type": "object",
+          "description": "Global configuration of log level",
+          "additionalProperties": false,
+          "properties": {
+            "level": {
+              "description": "Set between the different level options for log level",
```

```suggestion
              "description": "Defines logging severity levels.",
```
also specify, what happens in default value.